asmrun/Makefile.nt

```diff
@@ -13,62 +13,4 @@
 #*                                                                        *
 #**************************************************************************
 
-include ../config/Makefile
-
-include Makefile.shared
-
-CC=$(NATIVECC)
-CFLAGS=-I../byterun -DNATIVE_CODE -DTARGET_$(ARCH) -DSYS_$(SYSTEM) \
-       $(NATIVECCCOMPOPTS) $(IFLEXDIR)
-
-ifeq ($(TOOLCHAIN),mingw)
-ASMOBJS=$(ARCH).o
-else
-ASMOBJS=$(ARCH)nt.obj
-endif
-
-OBJS=$(COBJS) $(ASMOBJS)
-
-all: libasmrun.$(A)
-
-libasmrun.$(A): $(OBJS)
-	$(call MKLIB,libasmrun.$(A), $(OBJS))
-
-i386nt.obj: i386nt.asm
-	$(ASM)i386nt.obj i386nt.asm
```

Comment:
This indicates that this was probably written with msvc's command-line interface in mind. I'm not sure your new invocations are compatible with it however. To give you an idea on the possible matter:

    config/Makefile.mingw64:ASM=$(TOOLPREF)as
    config/Makefile.msvc64:ASM=ml64 -nologo -Cp -c -Fo